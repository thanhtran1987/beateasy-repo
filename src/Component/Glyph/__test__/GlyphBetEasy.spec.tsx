import * as React from 'react';
import { shallow } from 'enzyme';

import TestComponent from '../GlyphBetEasy';

describe('Tile', () => {
  it('should render correctly in default state', () => {
    const component = shallow(<TestComponent />);
    expect(component).toMatchSnapshot();
  });
});
