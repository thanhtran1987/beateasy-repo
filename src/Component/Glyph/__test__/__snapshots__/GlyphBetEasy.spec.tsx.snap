// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tile should render correctly in default state 1`] = `
<svg
  aria-hidden="true"
  height="32"
  viewBox="0 0 274 158"
  width="32"
>
  <defs>
    <linearGradient
      gradientTransform="matrix(1 0 0 1 -1.6539 0)"
      gradientUnits="userSpaceOnUse"
      id="logo-gradient"
      x1="151.5867"
      x2="270.7755"
      y1="123.8671"
      y2="4.6783"
    >
      <stop
        offset="0"
        stopColor="#23b4b7"
      />
      <stop
        offset=".8882"
        stopColor="#23b4b7"
        stopOpacity="0"
      />
    </linearGradient>
  </defs>
  <path
    d="M0 115.56V43.89h18.17c9.96 0 17.14 5.53 17.14 16.28v5.02c0 4.92-2.46 10.55-7.08 12.7 5.13 2.46 7.7 6.96 7.7 13.62v6.14c0 10.75-6.57 17.92-17.55 17.92H0zm12.32-41.67h3.7c4.52 0 6.98-3.17 6.98-7.37v-4.71c0-4.2-2.36-7.58-6.98-7.58h-3.7v19.66zm0 31.33h4.41c4.52 0 6.98-3.07 6.98-7.99V91.3c0-4.81-2.57-7.68-7.19-7.68h-4.21v21.6z"
  />
  <path
    d="M39.48 115.56V43.89H71.3v10.65H51.8v17.41h12.52V82.6H51.8v22.32h20.22v10.65H39.48z"
  />
  <path
    d="M98.28 115.56H86.06V54.53H74.97V43.89h34.39v10.65H98.28v61.02z"
  />
  <path
    d="M194.8 158c-43.67 0-79.2-35.44-79.2-79s35.53-79 79.2-79S274 35.44 274 79s-35.53 79-79.2 79zm0-145.33c-36.67 0-66.5 29.76-66.5 66.33s29.83 66.33 66.5 66.33 66.5-29.76 66.5-66.33-29.83-66.33-66.5-66.33z"
    fill="url(#logo-gradient)"
  />
  <path
    d="M144.76 109.09V48.91h20.81v6.96h-12.81v17.2h8.51v6.88h-8.51v22.18h13.16v6.96h-21.16z"
  />
  <path
    d="M193.51 109.09h-8.08l-1.2-11.52h-6.88l-1.29 11.52h-7.48l7.31-60.19h10.4l7.22 60.19zm-12.55-43.16l-2.92 25.54h5.59l-2.67-25.54z"
  />
  <path
    d="M217.5 69.03h-7.4v-9.54c0-2.92-1.12-4.38-3.44-4.38-2.06 0-3.27 1.63-3.27 3.96v5.42c0 3.61 1.2 6.53 3.53 8.77l5.33 5.07c3.96 3.7 5.93 8.68 5.93 15.13V98c0 6.96-4.39 11.69-11.52 11.69-7.14 0-11.35-4.39-11.35-11.26V86.82h7.48v11.61c0 2.67 1.29 4.39 3.7 4.39 2.32 0 3.61-1.63 3.61-4.13v-5.5c0-3.96-1.2-6.96-3.53-9.11l-4.99-4.64c-4.21-3.87-6.28-8.86-6.28-14.87V59.5c0-6.88 4.47-11.18 11.26-11.18 7.14 0 10.92 4.3 10.92 10.83v9.88z"
  />
  <path
    d="M235.72 109.09h-8V84.42l-9.11-35.51h8.68l4.47 22.87 4.9-22.87h8.17l-9.11 35.25v24.93z"
  />
</svg>
`;
